aikau/src/test/resources/alfresco/layout/DynamicHorizontalLayoutTest.js

```diff
@@ -283,6 +283,15 @@ define(["intern!object",
                   assert.closeTo(size.width, expectedWidth, 10, "Widget 5 was not the correct size");
                });
          },
+
+         "Check that when a widget is revealed that a resize event is published": function() {
+            return browser.findByCssSelector("body").end()
+               .clearLog()
+               .findById("HIDE_WIDGET_2_BUTTON_label")
```

Comment:
If this findById was first then you could call clearLog immediately after (and prior to the click) without affecting the CSS scope, and the find/end of the body would be redundant.